R/sqlOnly.R

```diff
@@ -0,0 +1,221 @@
+#' @file sqlOnly.R
+
+#' Internal function to create queries when running in SqlOnly mode
+#' 
+#' @noRd
+#' @keywords internal
+#' 
+.createSqlOnlyQueries <- function(
+    params,
+    check,
+    tableChecks,
+    fieldChecks,
+    conceptChecks,
+    sql,
+    connectionDetails,
+    checkDescription
+) {
+  qnum <<- qnum + 1
+  
+  check_description = SqlRender::render(
```

Comment:
to simplify this a bit, could you actually just pass in `params` here like they do in `runCheck`? (i didn't try this so not sure it would work but seems it might?)

Reply:
If OK, I'd rather keep this verbose so it is clear what parameters are passed.